### PR TITLE
Java Lambda tracing installation steps

### PR DESCRIPTION
**Pull request by Hesperide**
Edits Java Lambda function main installation steps to instrument Datadog distributed tracing for Lambda functions.

Staging: https://docs-staging.datadoghq.com/alex.cuoci/lambda-java-tracing-installation-steps/serverless/installation/java?tab=maven

### CODE REVIEW

File: content/en/serverless/installation/java.md [hunk 1]
```diff
@@ -13,21 +13,23 @@ further_reading:
...
 ## Required setup
 
 If not already configured:
 
 - Install the [AWS integration][1]. This allows Datadog to ingest Lambda metrics from AWS. 
 - Install the [Datadog Forwarder Lambda function][2], which is required to ingest AWS Lambda traces, enhanced metrics, custom metrics, and logs. 
 
-After you have installed the [AWS integration][1] and the [Datadog Forwarder][2], follow these steps to instrument your application to send metrics, logs, and traces to Datadog.
+After you have installed the [AWS integration][1] and the [Datadog Forwarder][2], follow these steps to instrument your application to send [enhanced Lambda metrics][3], logs, and traces to Datadog. 
+To fully instrument your serverless application with distributed tracing, your Java Lambda functions must be using the Java 8 Corretto (`Java8.al2`) or Java 11 (`Java11.al2`) runtimes.
```
Comment (agocs) [inline content/en/serverless/installation/java.md:26]:
```suggestion
To fully instrument your serverless application with distributed tracing, your Java Lambda functions must be using the Java 8 Corretto (`java8.al2`) or Java 11 (`java11`) runtimes.
```

File: content/en/serverless/installation/java.md [hunk 2]
```diff
@@ -80,25 +103,35 @@ dependencies {
...
-Although it's optional, Datadog highly recommends tagging you serverless applications with the `env`, `service`, and `version` tags following the [unified service tagging documentation][6].
+Although it's optional, Datadog highly recommends tagging you serverless applications with the `env`, `service`, and `version` tags following the [unified service tagging documentation][9].
 
 ## Explore Datadog serverless monitoring
 
-After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless Homepage][7].
+After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless Homepage][10].
 
-## Monitor custom business logic
+### Monitor custom business logic
```
Comment (agocs) [inline content/en/serverless/installation/java.md:127]: Should this be a sub-heading under "Explore Datadog Serverless Monitoring", or at an even level?

**Review by agocs — APPROVED** [review]
Looks good! just a couple of nits

File: content/en/serverless/installation/java.md (hunk 2 above)
Comment (Hesperide) [reply content/en/serverless/installation/java.md:127]: It's intentional, I'm actually surprised it wasn't already a sub-heading :)

File: content/en/serverless/installation/java.md [hunk 3]
```diff
@@ -80,25 +103,35 @@ dependencies {
 You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
 
 1. [Install the Datadog Forwarder if you haven't][2].
-2. [Subscribe the Datadog Forwarder to your function's log groups][5].
+2. [Subscribe the Datadog Forwarder to your function's log groups][6].
+
+### Monitor Java Lambda function cold starts
+
+Cold starts occur when your serverless applications receive sudden increases in traffic, and can occur when the function was previously inactive or when it was receiving a relatively constant number of requests. Users may perceive cold starts as slow response times or lag. Datadog highly recommends you configure a monitor on Java Lambda function cold starts, and use Datadog Serverless Insights to [keep cold starts to a minimum][7].
```
Comment (cswatt) [inline content/en/serverless/installation/java.md:110]:
```suggestion
Cold starts occur when your serverless applications receive sudden increases in traffic, including when the function was previously inactive or when it was receiving a relatively constant number of requests. Users may perceive cold starts as slow response times or lag. Datadog highly recommends you configure a monitor on Java Lambda function cold starts, and use Datadog Serverless Insights to [keep cold starts to a minimum][7].
```

File: content/en/serverless/installation/java.md [hunk 4]
```diff
@@ -80,25 +103,35 @@ dependencies {
...
+
+{{< img src="serverless/java-monitor-cold-starts.png" alt="Monitor Java Lambda Function Cold Starts"  style="width:100%;">}}
+
+To create a Datadog monitor on Java Lambda function cold starts, follow the monitor creation steps [here][8] with the following criteria:
```
Comment (cswatt) [inline content/en/serverless/installation/java.md:114]:
```suggestion
To create a Datadog monitor on Java Lambda function cold starts, follow the [monitor creation steps][8] with the following criteria:
```

File: content/en/serverless/installation/java.md [hunk 5]
```diff
@@ -112,26 +145,32 @@ public class Handler implements RequestHandler<APIGatewayV2ProxyRequestEvent, AP
...
-
         hc.connect();
+
+        ddl.finish();
     }
 }
 ```
 
-For more information on custom metric submission, see [here][8].
+For more information on custom metric submission, see [here][11].
```
Comment (cswatt) [inline content/en/serverless/installation/java.md:155]:
```suggestion
See the [custom metrics documentation][11] for more information on custom metric submission.
```

File: content/en/serverless/installation/java.md [hunk 6]
```diff
@@ -112,26 +145,32 @@ public class Handler implements RequestHandler<APIGatewayV2ProxyRequestEvent, AP
...
+
+### Connect logs and traces
+
+Follow the instructions [here][12] to automatically connect Java Lambda function logs and traces.
```
Comment (cswatt) [inline content/en/serverless/installation/java.md:159]:
```suggestion
To automatically connect Java Lambda function logs and traces, see the [Connecting Java Logs and Traces][12] for instructions.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
just some stylistic stuff

File: content/en/serverless/installation/java.md [hunk 7]
```diff
@@ -112,26 +145,32 @@ public class Handler implements RequestHandler<APIGatewayV2ProxyRequestEvent, AP
...
     }
 }
 ```
 
-For more information on custom metric submission, see [here][8].
+See the [custom metrics documentation][11] for more information on custom metric submission.
+
+### Connect logs and traces
+
+To automatically connect Java Lambda function logs and traces, see the [Connecting Java Logs and Traces][12] for instructions.
```
Comment (Hesperide) [inline content/en/serverless/installation/java.md:159]:
```suggestion
To automatically connect Java Lambda function logs and traces, see [Connecting Java Logs and Traces][12] for instructions.
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**